x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/list_common/empty_viewer_state/empty_viewer_state.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import type { FC } from 'react';
+
+import {
+  EuiLoadingContent,
+  EuiImage,
+  EuiEmptyPrompt,
+  EuiButton,
+  useEuiTheme,
+  EuiPanel,
+} from '@elastic/eui';
+import type { ExpressionColor } from '@elastic/eui/src/components/expression/expression';
+import type { EuiFacetGroupLayout } from '@elastic/eui/src/components/facet/facet_group';
+
+import illustration from '../../../../../common/images/illustration_product_no_results_magnifying_glass.svg';
+import { ViewerStatus } from '../types';
+import * as i18n from '../translations';
+
+interface EmptyViewerStateProps {
+  title?: string;
+  body?: string;
+
+  buttonText?: string;
+
+  listType?: string;
```

Comment:
`listType` throughout is usually referring to `ExceptionListTypeEnum` - might be easier to have it mapped out so user passes in a `ExceptionListTypeEnum` and we map it to what text it should be like:
```
const LIST_TYPE_TEXT: {
    [ExceptionListTypeEnum.ENDPOINT]: 'endpoint',
    [ExceptionListTypeEnum.DETECTION]: 'detection',
    [ExceptionListTypeEnum.RULE_DEFAULT]: 'rule',
};
```

That way anyone using this component doesn't need to know what text they should be using to refer to a specific exception for a certain list type.